apps/mobile/src/screens/ProfilePicturePickerScreen/ProfilePicturePickerScreen.tsx

```diff
@@ -41,6 +47,10 @@ export function ProfilePicturePickerScreen() {
   const [filter, setFilter] = useState<'Collected' | 'Created'>('Collected');
   const [networkFilter, setNetworkFilter] = useState<NetworkChoice>('all');
 
+  const screenTitle = useMemo(() => {
+    return currentScreen === 'ProfilePicture' ? 'Select a profile picture' : 'Select item to post';
```

Comment:
same comment about renaming from `ProfilePicturePickerScreen` : feels out of place for `ProfilePicturePickerScreen` to contain logic about showing a header saying "Select item to post"